src/main/java/org/generationcp/middleware/api/location/LocationService.java

```diff
@@ -36,4 +36,8 @@
 	 * @return the favorite project location ids
 	 */
 	List<Integer> getFavoriteProjectLocationIds(String programUUID);
+
+	long countLocations(LocationSearchRequest locationSearchRequest);
```

Comment:
Why do we need new method `countLocations` instead of using existing `countFilteredLocations`?

Reply:
No need for `countLocations`, replace usage with countFilteredLocations